### ISSUE TITLE
[BUG] Entry content is being appended after video players from extensions

### ISSUE THREAD

**Issue by bwees** [FreshRSS/FreshRSS#5563]
**Describe the bug**
The feed content is appended to the end of the view when using an extension like https://github.com/kevinpapst/freshrss-youtube
This bug appears in v1.21.0 and not in v1.20.2

I have raised the following issue on the extension's page as well
https://github.com/kevinpapst/freshrss-youtube/issues/21

**To Reproduce**
1. Install the https://github.com/kevinpapst/freshrss-youtube extension on 1.21.0
2. Load a youtube channel RSS feed
3. View an entry

**Expected behavior**
You will see the feed entry content, thumbnail, and save icon which the youtube extension is meant to remove. The video player will load but it does not replace the entry contents like it should.

**Screenshots:**

v1.20.2 (what its supposed to look like):
<img width="302" alt="Screenshot 2023-08-03 at 1 23 03 PM" src="https://github.com/FreshRSS/FreshRSS/assets/12686250/3815d331-03db-4d21-a4ae-b70361f583b2">

v1.21.0 (what it actually looks like):
![image](https://github.com/FreshRSS/FreshRSS/assets/12686250/b4fb6d87-0b83-4944-a51e-a2d7114dce9f)

**Environment information (please complete the following information):**
- Device: x86 box
- OS: Docker Container
- Browser: Chrome Latest
- FreshRSS version: 1.21.0 (issue is not present in 1.20.2)
- Database version: SQLite
- PHP version: unknown
- Installation type: Docker

**Comment by kevinpapst** [comment]
Repeating my statement from my repo:

The setting worked properly in the past. You either saw the video player alone or (if setting was active) the player and the content below. 

The thumbnail is new, also the fact that the content is injected, no matter what.
There was some kind of change in FreshRSS core.

**Comment by bwees** [comment]
I will do a diff on the entry model to see if something got added by accident

**Comment by kevinpapst** [comment]
Maybe the order of plugin execution changed? Honestly: you see that my code was last changed 5 years ago. It is impressive that it still works and the core team has such a strong BC policy. 

Probably @Alkarex has an idea what might have caused that behavior change?

**Comment by bwees** [comment]
Yeah it seems odd that the plugin is still able to push data to the html view but the article is then appended. I wonder if entry._content got unexpectedly deprecated

**Comment by bwees** [comment]
I haven’t dived into it yet but this PR may be the culprit. https://github.com/FreshRSS/FreshRSS/pull/4944

**Comment by math-GH** [comment]
yes, there was an improvement with the attachments in 1.21.0

**Comment by pax0707** [comment]
Any update on this?
